src/eos-updater-apply.c

```diff
@@ -78,7 +78,6 @@ apply (GTask *task,
   GMainContext *task_context = g_main_context_new ();
   const gchar *update_id = eos_updater_get_update_id (updater);
   const gchar *update_refspec = eos_updater_get_update_refspec (updater);
-  const gchar *orig_refspec = eos_updater_get_original_refspec (updater);
```

Comment:
Hm, I remember seeing in a review that introduced those changes that exporting this property was just a side-effect of adding a property which was meant to be used only internally (read: there was no `EosUpdaterData` struct then, which has this very purpose).

But sure, if you think that exporting this property might be useful for client then I'll just remove this commit.